<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterService</class>
 <widget class="QWidget" name="RegisterService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>463</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>676</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QLabel" name="labelimage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>67</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>67</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>个性签名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditsignature">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 226, 52);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>8</y>
     <width>262</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditname">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxsex">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string>出生如期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>67</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string>真实姓名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditimagepath">
      <property name="minimumSize">
       <size>
        <width>361</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>361</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonpath">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string>......</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>428</width>
     <height>269</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditUser">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditpassword">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditsure">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>户籍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAddress">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIdcard">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdittel">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditemil">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
